provider/manual/environ.go

```diff
@@ -42,7 +43,7 @@ const (
 
 var (
 	logger                 = loggo.GetLogger("juju.provider.manual")
-	manualCheckProvisioned = manual.CheckProvisioned
+	manualCheckProvisioned = linux.CheckProvisioned
```

Comment:
I think something needs to be reworked here; the abstraction (poor as it was: global variable) doesn't seem to fit any longer.